### ISSUE TITLE
`show ranges from table` could be serving a user with a misleading info

### ISSUE THREAD

**Issue by a-entin** [cockroachdb/cockroach#82995]
**Describe the problem**

TLDR; A user may be intuitively expecting all ranges that have keys related to the table. But it does not include ranges for the secondary indexes.

More:

`show ranges from table foo` is exactly equivalent to `show ranges from index foo@primary`.
I.e. it does not include `show ranges from index foo@secondary_idx`.  Seems counterintuitive and most likely not what a user would expect.

Suggested resolution:
`show ranges from table` should return a superset of all ranges across all indexes, not just PK. Otherwise a justification for having  `show ranges from table` when there is `show ranges from index` is not clear.

Jira issue: CRDB-16785
